src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -489,16 +489,14 @@ class TcgStatics {
     if(card.getCardTypes().is(BREAK)){
       bg.wcu("BREAK Pokémon cannot be brought to play")
     }
-    PlayerType playerType=targetPlayer.getPlayerType(bg())
     PlayerBattleground pbg=targetPlayer.getPbg(bg())
     assert pbg.bench.isNotFull() : "Bench is full"
```

Comment:
This check should be done in the new PutOnBench implementation rather than here. It should then throw an `EffectRequirementException`.

Reply:
I've moved the effect over to PutOnBench